cocotb/scheduler.py

```diff
@@ -628,50 +631,71 @@ async def wrapper():
 
         return wrapper()
 
-    def add(self, coroutine):
-        """Add a new coroutine.
-
-        Just a wrapper around self.schedule which provides some debug and
-        useful error messages in the event of common gotchas.
-        """
+    @staticmethod
+    def create_task(coroutine: Any) -> RunningTask:
+        """ Checks to see if the given object is a schedulable coroutine object """
```

Comment:
```suggestion
        """ Checks to see if the given object is a schedulable coroutine object and if so, returns it """
```
? Otherwise, "create" and "Checks" do not fit IMO.